hw/xbox/nv2a.c

```diff
@@ -1283,7 +1285,11 @@ static const ColorFormatInfo kelvin_color_format_map[66] = {
         {2, false, GL_RGB8_SNORM, GL_RGB, GL_BYTE}, /* FIXME: This might be signed */
     [NV097_SET_TEXTURE_FORMAT_COLOR_SZ_G8B8] =
         {2, false, GL_RG8_SNORM, GL_RG, GL_BYTE, /* FIXME: This might be signed */
-         {GL_RED, GL_GREEN, GL_RED, GL_GREEN}},
+         {GL_ZERO, GL_RED, GL_GREEN, GL_ONE}},
+    [NV097_SET_TEXTURE_FORMAT_COLOR_SZ_R8B8] =
+        {2, false, GL_RG8_SNORM, GL_RG, GL_BYTE, /* FIXME: This might be signed */
```

Comment:
No, there is no GL_RB. GL_RG will be 2 channels - we use a swizzle mask to reorder the channels afterwards [R source is R, G will be 0.0, B source is G, A will be 1.0].
So even if the texture is loaded to Red and Green { Red, Green, ?, ? } it will be sampled as { Red[=Red], 0.0, Blue[=Green], 1.0 } which is probably what R8B8 does.